### ISSUE TITLE
On books page, keyboard navigation with tab key skips the element that should not be skipped.

### ISSUE THREAD

**Issue by bick-jp** [internetarchive/openlibrary#7858]
You cannot focus on "Read more" on the books page via keyboard tab-key navigation.

### Evidence / Screenshot (if possible)
- The focus is on "English" now.
<img width="795" alt="Screen Shot 2023-05-11 at 2 14 22 PM" src="https://github.com/internetarchive/openlibrary/assets/8709562/5ce3fd5b-8741-4228-838f-286331dda116">

- If I press the tab key, it will skip the "Read more" and go to the next focusable element, which is "Fantasy" in this case.
<img width="793" alt="Screen Shot 2023-05-11 at 2 14 28 PM" src="https://github.com/internetarchive/openlibrary/assets/8709562/1f14de9d-f745-4a38-bc72-706f2e4a1563">

### Relevant url?

The example above is from [A Court of Mist and Fury](https://openlibrary.org/works/OL17860744W/A_Court_of_Mist_and_Fury#details), but I think this happens to any other books pages.

### Steps to Reproduce

1. Go to [A Court of Mist and Fury](https://openlibrary.org/works/OL17860744W/A_Court_of_Mist_and_Fury#details)
2. Keep pressing the tab key until you reach the point described in the example above.

* Actual: It skips "Read more"
* Expected: "Read more" should be focused.

### Details

- **Logged in (Y/N)?** Y
- **Browser type/version?** Google Chrome Version 113.0.5672.63 (Official Build) (arm64). I also checked with Safari Version 15.4 (17613.1.17.1.13).
- **Operating system?** macOS Monterey Version 12.3.1
- **Environment (prod/dev/local)?** prod

**Comment by natea123** [comment]
Hello! I would like to work on this bug if it is still unassigned.
